{{cookiecutter.project_name}}/examples/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.10.2)
+project({{ cookiecutter.module_name }}Examples)
```

Comment:
Do we need to add `CXX` or does this help CMake somehow exclude any Python example file that is added along?

Reply:
Python examples don't need to be built -- the default include `CXX` -> this is good.

Reply:
:thinking:  So do we need to use
```
project({{ cookiecutter.module_name }}Examples CXX)
```
explicitly then?